### PR TITLE
Docker 1.11.1 && docker-compose 1.7.0

### PR DESCRIPTION
**Pull request by caiofbpa**
I think I'm having some issues regarding docker-compose naming because the agent uses 1.6.0 and I use 1.7.0.

### CODE REVIEW

**Comment by lox** [comment]
Thanks for the PR! Would it be possible to add this to the https://github.com/buildkite/docker-buildkite-agent/blob/master/scripts/list.sh as well so that we get the newer versions in ubuntu and alpine images?

**Comment by caiofbpa** [comment]
@lox: Done!

File: scripts/list.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ DOCKER_VERSIONS=(
   1.8.3
   1.9.1
   1.10.1
```
Comment (lox) [inline scripts/list.sh]: This should be bumped to 1.10.3, this file contains the latest point release of each major version

**Comment by lox** [comment]
Also, looks like 1.11.1 was released :)

**Comment by caiofbpa** [comment]
@lox: updated the Docker `1.10.1` to `1.10.3` and the `1.11.0` to `1.11.1`

Is there anything else I can help with?

**Comment by lox** [comment]
Looks good to me! Shall we merge this @toolmantim?

**Comment by toolmantim** [comment]
Yep!

**Comment by lox** [comment]
Urgh this is failing builds due to https://forums.docker.com/t/docker-apk-package-for-alpine-linux-has-an-unresolved-dependency-to-libseccomp/9604

**Comment by caiofbpa** [comment]
I found it weird that Buildkite didn't run any builds for this PR. Is there anything I could do to help fix that libseccomp issue?

**Comment by keithpitt** [comment]
@caiofbpa we actually don't run builds for third party code just yet, so it's only branches on the main repo that get builds. We want to fix this though!!

**Pull request merged**